rokka/client.go

```diff
@@ -13,16 +15,25 @@ type Client struct {
 type Config struct {
 	APIAddress string
 	APIVersion string
-	APIKey string
+	APIKey     string
 	HTTPClient *http.Client
 }
 
+type StatusCodeError struct {
+	StatusCode int
+	Body       []byte
+}
+
+func (e StatusCodeError) Error() string {
+	return fmt.Sprintf("Status Code %d", e.StatusCode)
+}
```

Comment:
go standard library errors prefix errors with the package name usually. In this case: `return fmt.Sprintf("rokka: Status Code %d", e.StatusCode)`.

The other option would be (and I think we should actually do that): Use a package like https://github.com/pkg/errors to encapsulate the errors we encounter. This way, consumers can freely decided what kind of error they need to react correctly.

Reply:
done